### PR TITLE
helper udev script added && keebie.service improved

### PR DESCRIPTION
**Pull request by parasbhanot**
i have added the udev script which will automatically launch the keebie.py and it will give the read/write permission to the target keyboard when it is plugged in . 

keebie.service is also improved : basically, we need to add "Environment=XAUTHORITY=/home/$USER/.Xauthority" in your service file so that systemd can start is without any problem . Without setting the Environment variable service fails to start in arch /Majaro linux.

### CODE REVIEW

**Comment by Michael-E-B** [comment]
I know next to nothing about udev or systemd, but it seems to me as though your `setup_tools/udevRule.sh` script is targeted at one keyboard in particular. Is there a reason for this?

**Comment by parasbhanot** [comment]
The udevRule.sh  is generic script. You have to replace  `ATTRS{idVendor}=="0c45", ATTRS{idProduct}=="5004"` in udevRule.sh  with vid and pid of your second keyboard which you can find using `lsusb `command.

udev (userspace /dev) is a device manager for the Linux kernel . udev also handles all user space events raised when hardware devices are added into the system or removed from it. 

So in this case i have written the udev rule in "**udevRule.sh"** which automatically gives 644 (read/write) permission to your second keyboard  and starts the  "keebie.service" which in turn runs keebie.py when you plug the second keyboard to your usb port. 

In summary : once udev Rule is made for your second keyboard you do not have to use `sudo chmod a+rw /dev/input/by-id/[my-device-name]` every time we plug/unplug the second keyboard. 

Since these day  linux distros like arch , ubuntu/debian, opensuse, fedora  use  systemd as their init system so it is better to use keebie.service instead of using & or  forceBackground option in keebie.py . The main advantage of running keebie as systemd service is that you can start and stop the keebie.py using` systemctl start/stop/status keebie`.

**Comment by daisyUniverse** [comment]
These changes look okay to me and the new service file is an improvement over the current one, I'll go ahead and merge
